### PR TITLE
Update information about supported and deprecated OS and Java

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1996.surge.sh

Updated pages:
- [System requirements](https://neo4j-docs-operations-1996.surge.sh/operations-manual/5/installation/requirements)

**Review by renetapopova — APPROVED** [review]
Looks good

File: modules/ROOT/pages/installation/requirements.adoc [hunk 1]
```diff
@@ -108,9 +108,19 @@ For cloud environments, and server-based, on-premise environments:
...
+* MacOs 11, 12
+* Windows Desktop 10
+* Amazon Linux 2022 AMI
+* Unbuntu Server 16.04, 18.04, 20.04
+* Windows Server 2016, 2019
+
 [NOTE]
 ====
-From version 5.14 onwards, Neo4j also supports JDK 21 on all Operating Systems.
+The Neo4j 5.26 LTS is the last version that uses the Java SE 17 Platform.
```
Comment (renetapopova) [inline modules/ROOT/pages/installation/requirements.adoc:122]: I think this note is good but should be moved to the Java section.

Comment (NataliaIvakina) [reply modules/ROOT/pages/installation/requirements.adoc:122]: good

File: modules/ROOT/pages/installation/requirements.adoc [hunk 2]
```diff
@@ -147,6 +157,7 @@ The minimum requirement is Java Runtime Environment (JRE).
 | 4.x           | Java SE 11 Platform Specification
 | 5.x           | Java SE 17 Platform Specification
 | 5.14          | Java SE 17 and Java SE 21 Platform Specification
+| 5.26 LTS      | This is the last version that uses the Java SE 17 Platform. It is recommended to move to Java SE 21.
```
Comment (renetapopova) [inline modules/ROOT/pages/installation/requirements.adoc:160]: And here maybe we can remove this line and just have the note.
```suggestion
```

File: modules/ROOT/pages/installation/requirements.adoc [hunk 3]
```diff
@@ -108,9 +108,19 @@ For cloud environments, and server-based, on-premise environments:
 | *Windows Server 2016, 2019, 2022*                | OracleJDK 17, ZuluJDK 17
 |===
 
+label:deprecated[Deprecated in 5.26]
```
Comment (renetapopova) [inline modules/ROOT/pages/installation/requirements.adoc:111]: This is good for now, but when we have the versions of the OSs that we actually support, we can update the tables above instead.

**Review by renetapopova — COMMENTED** [review]
Sorry, I didn't look at the preview when I approved it the first time. But after I looked, I had a couple of comments.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1996.surge.sh

Updated pages:
- [System requirements](https://neo4j-docs-operations-1996.surge.sh/operations-manual/5/installation/requirements)

**Pull request merged**